CDAP-8166 Add notes and notice of dropping support

awholegunch opened the pull request:
Fix for https://issues.cask.co/browse/CDAP-8166

Running as Quick Build: http://builds.cask.co/browse/CDAP-DQB258-4

Page of interest: http://builds.cask.co/artifact/CDAP-DQB258/shared/build-4/Docs-HTML/4.1.0-SNAPSHOT/en/admin-manual/hadoop-compatibility.html

Commit:
Add notes and notice of dropping support.

sreevatsanraman commented on cdap-docs/admin-manual/source/hadoop-compatibility.rst:
```diff
@@ -122,11 +123,16 @@ Hadoop components, as required:
+
+.. _admin-manual-hadoop-compatibility-matrix-note-4:
+
+**Note 4:** An upcoming release of CDAP will drop support for all versions
```
Why don't we say which release and why?

bdmogal replied:
+1.

Commit:
Merge branch 'release/4.1' into docs/r4.1.0_8166_drop_support

# Conflicts:
#	cdap-docs/admin-manual/source/hadoop-compatibility.rst
# Resolved by editing.

Commit:
Remove definitions.

Commit:
Added version.

sreevatsanraman commented on cdap-docs/admin-manual/source/hadoop-compatibility.rst:
```diff
@@ -100,30 +102,35 @@ Hadoop components, as required:
+.. _admin-manual-hadoop-compatibility-matrix-note-4:
+
+**Note 4:** An upcoming release of CDAP (scheduled for CDAP 4.3) will drop support for all
+versions older than *CDH 5.4.11* or *HDP 2.5.0.0*.
```
we should mention why as well. @bdmogal what do you think?

bdmogal replied:
👍 Ideally it would be nice if there was a link to an announcement from the distros dropping support, too.

awholegunch replied:
Best I was able to find was to add links to the "Apache Hadoop Privilege Escalation Vulnerability" and links to either the distro's explanation (Cloudera) or the relevant
distro (Hortonworks) with the fixed version.

Commit:
Add links to the "Apache Hadoop Privilege Escalation Vulnerability" and
the distress, either links to their explanation (Cloudera) or the relevant
distro (Hortonworks) with the fixed version.

sreevatsanraman commented:
Lgtm

bdmogal commented on cdap-docs/admin-manual/source/hadoop-compatibility.rst:
```diff
@@ -100,30 +102,40 @@ Hadoop components, as required:
+**Note 4:** An upcoming release of CDAP (scheduled for CDAP 4.3) will drop support for all
+versions older than `CDH 5.4.11
+<https://www.cloudera.com/documentation/other/security-bulletins/topics/csb_all_product_issues.html#tsb_175>`__
+or `HDP 2.5.0.0 
```
sorry for not catching this earlier.are we really dropping support for everything below 2.5.x? Does that mean we would support only 1 HDP minor release?

awholegunch replied:
Yes.

wolf31o2 replied:
Hortonworks has chosen to not backport security patches into their prior releases.
